internal/commands/app/diff.go

```diff
@@ -229,7 +240,7 @@ func (i *diffInputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 		i.LocalPath = app.RootDir
 	}
```

Comment:
thoughts on adding an `else` condition here to error out if we _still_ don't have a local path which points to an app?  this appears to be an existing bug but it seems like we can still end up in a situation where `i.LocalPath` and `app.RootDir` is empty, which would end up failing the first line of the diff handler when we load the app

if we do go that route, i'd probably flip the if condition (but am commenting here for letting this comment become "Outdated"):

```go
	if app.RootDir == "" {
		return errProjectExistsNot(i.LocalPath) // i'll let you pick a better name
                // it actually looks like we could modify the errInvalidPath you added to account for both
                // a path and project not existing (similar to how errProjectInvalid works in the push package)
	} else {
                i.LocalPath = app.RootDir
        }
```